### PR TITLE
removed placeholder

### PR DESCRIPTION
**Pull request by smptr0911**
removed all the placeholders and unused files

### CODE REVIEW

**Review by dbigueta — CHANGES_REQUESTED** [review]
Can you make the `import App from '/App.jsx';` as line 1, just to shift the order of imports alphabetically

File: src/main.jsx [hunk 1]
```diff
@@ -1,10 +1,9 @@
-import { StrictMode } from 'react'
-import { createRoot } from 'react-dom/client'
-import App from './App.jsx'
-import './index.css'
+import App from './App.jsx';
+import { StrictMode } from 'react';
+import { createRoot } from 'react-dom/client';
```
Comment (dbigueta) [inline src/main.jsx]: Can you also switch the order of these 2 thanks!

**Review by dbigueta — CHANGES_REQUESTED** [review]
One last change!

File: src/main.jsx (hunk 1 above)
Comment (smptr0911) [reply src/main.jsx]: did it!

**Review by dbigueta — APPROVED** [review]
I love it!

**Pull request merged**